config/prod/tom-cvat.yaml

```diff
@@ -15,7 +15,7 @@ parameters:
   # You may omit them if you do not need to override the defaults.
 
   # (Optional) EC2 instance type, default is "t2.nano" (other available types https://aws.amazon.com/ec2/instance-types/)
-  InstanceType: "t2.xlarge"
+  InstanceType: "p3.2xlarge"
```

Comment:
This instance is not supported in the managed-ec2-v7.yaml file.  you will need to update that file first. https://github.com/Sage-Bionetworks/aws-infra/blob/master/templates/managed-ec2-v7.yaml

Reply:
added support here: https://github.com/Sage-Bionetworks/aws-infra/pull/139